joplin/pages/event_page/models.py

```diff
@@ -67,10 +85,22 @@ class EventPage(JanisBasePage):
                         ('additional_details_vi', TextBlock(label='Any other necessary location details, such as room number [vi]', required=False)),
                     ],
                 )),
+                ('virtual_event', StructBlock(
+                    [
+                        ('event_link', TextBlock(label='Event link or location')),
+                        ('additional_information_en', TextBlock(label='Any other necessary information, such as '
+                                                                      'meeting code [en]', required=False)),
+                        ('additional_information_es', TextBlock(label='Any other necessary information, such as '
+                                                                      'meeting code [es]', required=False)),
+                        ('additional_information_ar', TextBlock(label='Any other necessary information, such as '
+                                                                      'meeting code [ar]', required=False)),
+                        ('additional_information_vi', TextBlock(label='Any other necessary information, such as '
+                                                                      'meeting code [vi]', required=False)),
+                    ]))
```

Comment:
There might be some work to do updating the EventPageLocation node in api/schema.py to accommodate the new virtual_event location type.